pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/QuestionDao.java

```diff
@@ -1593,6 +1643,13 @@ default void insertQuestion(long activityId, AgreementQuestionDef agreementQuest
         }
     }
 
+    default boolean deleteDateQuestion(long questionId) {
+        getJdbiDateQuestionFieldOrder().deleteForQuestionId(questionId);
+        var val = getJdbiDateQuestion().delete(questionId);
+        LOG.info("Deleted date question id? {}", val);
```

Comment:
The question ID is missing from the log statement.

```suggestion
        LOG.info("Deleted date question id {}? {}", questionId, val);
```